shell/browser/ui/tray_icon_gtk.cc

```diff
@@ -4,21 +4,44 @@
 
 #include "shell/browser/ui/tray_icon_gtk.h"
 
+#include "base/optional.h"
 #include "base/strings/stringprintf.h"
 #include "base/strings/utf_string_conversions.h"
 #include "chrome/browser/ui/views/status_icons/status_icon_linux_dbus.h"
 #include "shell/browser/browser.h"
 #include "shell/common/application_info.h"
 #include "ui/gfx/image/image.h"
+#include "ui/gfx/image/image_skia_operations.h"
 
 namespace electron {
 
 TrayIconGtk::TrayIconGtk() = default;
 
 TrayIconGtk::~TrayIconGtk() = default;
 
+gfx::ImageSkia GetIconFromImage(const gfx::Image& image) {
```

Comment:
Utility functions should be put in anonymous namespace to avoid name conflicts.

An example can be found at:
https://cs.chromium.org/chromium/src/base/guid.cc?type=cs&g=0&l=16-45

Reply:
Ah good catch!

Reply:
agreed, nice catch. fixed 418de05.